common/src/main/java/com/microsoft/identity/common/internal/servertelemetry/Schema.java

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.servertelemetry;
+
+import com.microsoft.identity.common.internal.telemetry.TelemetryEventStrings;
+import com.microsoft.identity.common.internal.util.StringUtil;
+
+import java.util.Arrays;
+
+/**
+ * This class defines the schema for server-side telemetry
+ */
+public class Schema {
+
+    public static final String CURRENT_REQUEST_HEADER_NAME = "x-client-current-telemetry";
+    public static final String LAST_REQUEST_HEADER_NAME = "x-client-last-telemetry";
+
+    public static final class Key {
+        // new keys
+        public static final String SCHEMA_VERSION = "schema_version";
+
+        //imported keys
+        public static final String API_ID = TelemetryEventStrings.Key.API_ID;
+        public static final String FORCE_REFRESH = TelemetryEventStrings.Key.IS_FORCE_REFRESH;
+        public static final String CORRELATION_ID = TelemetryEventStrings.Key.CORRELATION_ID;
+        public static final String ERROR_CODE = TelemetryEventStrings.Key.ERROR_CODE;
+        public static final String ACCOUNT_STATUS = TelemetryEventStrings.Key.ACCOUNT_STATUS;
+        public static final String ID_TOKEN_STATUS = TelemetryEventStrings.Key.ID_TOKEN_STATUS;
+        public static final String AT_STATUS = TelemetryEventStrings.Key.AT_STATUS;
+        public static final String RT_STATUS = TelemetryEventStrings.Key.RT_STATUS;
+        public static final String FRT_STATUS = TelemetryEventStrings.Key.FRT_STATUS;
+        public static final String MRRT_STATUS = TelemetryEventStrings.Key.MRRT_STATUS;
+    }
+
+    public static final class Value {
+        public static final String SCHEMA_VERSION = "1";
```

Comment:
Two variables named `SCHEMA_VERSION` in this file is confusing -- I suggest making this harder to harder to accidentally use the wrong value by renaming these to something like:
`SCHEMA_VERSION` and `VERSION_1`

Consider what happens when the version changes to `2` in the future, now you'll be changing the value of this constant, rather than adding a new one -- if you decide you want to clean up the v1 cache when you rev to v2 you'll end up hardcoding `1` again under a different variable name so might as well plan to have a few schema versions

Reply:
Addressed as discussed offline.